SYCL/InvokeSimd/call_vadd_1d.cpp

```diff
@@ -0,0 +1,195 @@
+// TODO: enable on Windows once driver is ready
```

Comment:
I think this test is a duplicate of invoke_simd_smoke.cpp

Reply:
Agree, test removed.